Jenkinsfile.d/core/package

```diff
@@ -246,55 +247,53 @@ pipeline {
             }
           }
         }
-        stage('Windows') {
-          when {
-            environment name: 'WINDOWS_PACKAGING_ENABLED', value: 'true'
-          }
-          stages {
-            stage('Build') {
-              // Windows requirement: Every steps need to be executed inside default jnlp
-              // as the step 'container' is known to not be working
-              agent {
-                kubernetes {
-                  yamlFile 'PodTemplates.d/package-windows.yaml'
-                }
-              }
-              steps {
-                container('dotnet') {
-                  checkout scm
-                  dir (WORKING_DIRECTORY){
-                    git branch: PACKAGING_GIT_BRANCH, credentialsId: 'release-key', url: PACKAGING_GIT_REPOSITORY
-
-                    unstash 'GPG'
-                    unstash 'WAR'
-                    unstash 'KEYSTORE'
-
-                    powershell '''
-                      Get-ChildItem env:
-                      $env:WAR=(Resolve-Path .\\jenkins.war).Path
-                      & .\\make.ps1
-                    '''
-                    // Don't archive the full path
-                    dir('msi\\build\\bin\\Release\\en-US') {
-                      archiveArtifacts '*.msi*'
-                    }
-
-                  }
-                }
-              }
-            }
-            stage('Publish'){
-              steps {
-                unarchive mapping: ['*.msi*': WORKING_DIRECTORY]
-                sshagent(['pkgserver']) {
-                  sh '''
-                    ./utils/release.bash --packaging msi.publish
-                  '''
-                }
-              }
-            }
-          }
-        }
+        // TODO: Enable Windows packaging again when code signing certificate is available
+        // stage('Windows') {
+        //   when {
+        //     environment name: 'WINDOWS_PACKAGING_ENABLED', value: 'true'
```

Comment:
Seems overkill to comment all of this out when it is keyed off `WINDOWS_PACKAGING_ENABLED == true` (which you are setting to false above). Not a blocking comment though.